ArduCopter/control_autotune.cpp

```diff
@@ -684,25 +684,25 @@ void Copter::autotune_attitude_control()
         switch (autotune_state.axis) {
         case AUTOTUNE_AXIS_ROLL:
             if ((autotune_state.tune_type == AUTOTUNE_TYPE_SP_DOWN) || (autotune_state.tune_type == AUTOTUNE_TYPE_SP_UP)) {
-                rotation_rate = rotation_rate_filt.apply(direction_sign * (ToDeg(ahrs.get_gyro().x) * 100.0f), MAIN_LOOP_SECONDS);
+                rotation_rate = rotation_rate_filt.apply(direction_sign * (ToDeg(ahrs.get_gyro().x) * 100.0f), scheduler.get_loop_period_s());
```

Comment:
I think it would be better to do it in a macro than to call a method. To avoid saving and loading the stack.

Reply:
Point is that this is run-time-settable, not a compile-time-constant.